tools/alluxio/tarball.sh

```diff
@@ -1,12 +1,12 @@
 set -x -e
 
 # download go package if not exist
-[ -e "go1.13.6.linux-amd64.tar.gz" ] \
-  || wget https://studygolang.com/dl/golang/go1.13.6.linux-amd64.tar.gz
+[ -e "go1.18.1.linux-amd64.tar.gz" ] \
```

Comment:
We upgraded the tarball generation to use go1.18.1: https://github.com/Alluxio/alluxio/pull/15881